archetypes/oci-mp/src/main/resources/server/README.md.mustache

```diff
@@ -0,0 +1,393 @@
+# Helidon OCI Archetype Server using MP
+
+This module demonstrates a simple Hello World JAX-RS based application and is composed of the following:
+
+1. Server-side source code that will be generated using OpenApi tool based on an OpenApi specification document in the form
+   of [api.yaml](../spec/api.yaml) as an input. The generated source code will be located in
+   [target/generated-sources/server/src/main/java/__pkg__/server](target/generated-sources/server/src/main/java/__pkg__/server)
+   and will have the following components:
+   * A JAX-RS base interface
+   * A model source code that will represent a JSON object
+2. A server application that will implement the generated JAX-RS base interface to add a simple hello world
+greet logic. Furthermore, it will also include OCI SDK integration with OCI Logging and Metrics.
+3. It also showcases how to use generated [client](../client) to test this server application.
+   
+## OCI SDK integration
+Helidon provides easy integration with the OCI SDK by injecting SDK clients. This server application provides a sample
+demonstration for both the OCI Monitoring and Logging SDK in [GreetResource.java](src/main/java/__pkg__/GreetResource.java) with simple annotations like below:
+```java
+    // OCI SDK integration of monitoring and logging
```

Comment:
Please use constructor injection in examlpes and production classes - this simplifies unit testing, and also does not break visibility restrictions of JVM

Reply:
Updated to use constructor injection.